irods_catalog_provider/entrypoint.sh

```diff
@@ -11,13 +11,11 @@ done
 
 echo "iRODS catalog database is ready"
 
-setup_input_file=/irods_setup.input
+echo "Preparing hostnames"
+sed -i "s/THE_HOSTNAME/${HOSTNAME}/g" /unattended_install.json
 
-if [ -e "${setup_input_file}" ]; then
-    echo "Running iRODS setup"
-    python3 /var/lib/irods/scripts/setup_irods.py < "${setup_input_file}"
-    rm /irods_setup.input
-fi
```

Comment:
I think we need to keep something like this around. With these changes, the entrypoint script will attempt to setup the server every time the container starts (I think), which of course will only work the first time. Can you confirm this?

If that's true, we can do basically the same thing, keying off of the `/unattended_install.json` file instead of `/irods_setup.input`.

Reply:
Hmm, well, the /unattended_install.json will always be there... in its current state.

We could easily produce a *new* json via the `sed` replacement... and then we can use the 'new' file as the conditional for the initial logic...

Reply:
The current input file is acting both as the input for the setup script and as an indicator that setup needs to be run. When the image is built, we `COPY` the file in and the entrypoint will run setup the first time it starts because the input file exists, and then it removes the input file afterwards which means setup will not be run again (unless we intentionally create that file in that container again).

We can now do the same thing for the unattended JSON input file. In that way, the JSON file will only be there the first time... We would remove it after setup completes. Similarly, the sed replacement could happen inside the if statement, only the first time. We wouldn't need to any of it after that.